Open Judge System/Data/OJS.Data/Migrations/201708220830272_identity_update.cs

```diff
@@ -0,0 +1,56 @@
+namespace OJS.Data.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+
+    public partial class identity_update : DbMigration
```

Comment:
for future reference - you might want to use `PascalCase` when naming Migrations - this is a `.cs` file in its essence. CS files we name in PascalCase.